configure.ac

```diff
@@ -1310,9 +1315,9 @@ if [[ "$a8_target" = "windx" ]]; then
 fi
 if [[ "$SUPPORTS_RDEVICE" = "yes" ]]; then
     echo "Using R: device?......................: $WANT_R_IO_DEVICE"
-        if [[ "$a8_host" != "win" -a "$WANT_R_IO_DEVICE" = "yes" ]]; then
-            echo "Using R: with the host serial port?...: $WANT_R_SERIAL"
-            echo "Using R: with IP network support......: $WANT_R_NETWORK"
+        if [[ "$a8_host" != "win" -a "$a8_host" != "macos" -a "$WANT_R_IO_DEVICE" = "yes" ]]; then
```

Comment:
I know you have been just following the suit here but it would be nice to move this `$a8_host` check into the `dnl Select host/target specific features...` section.

Reply:
There are a8_host checks littered through the entire file. I agree this is something that should be cleaned up but its a larger task than just getting the mac build to work without specifying configure options. 

Suggest that gets raised as another issue for someone to pick up at a later date.